### ISSUE TITLE
[Bug]: Gimp & Inkscape could not be installed

### ISSUE THREAD

**Issue by hansm629** [termux/termux-packages#21904]
### Problem description

Suddenly, the installation is not working. Please check.

### What steps will reproduce the bug?

```
$ pkg install gimp inkscape
No mirror or mirror group selected. You might want to select one by running 'termux-change-repo'
Checking availability of current mirror:
[*] https://packages-cf.termux.org/apt/termux-main: ok
Reading package lists... Done
Building dependency tree... Done
Reading state information... Done
Some packages could not be installed. This may mean that you have
requested an impossible situation or if you are using the unstable
distribution that some required packages have not yet been created
or been moved out of Incoming.
The following information may help to resolve the situation:

The following packages have unmet dependencies:
 libassuan : Breaks: gpgme (< 2.4.5-3) but 1.23.2-3 is to be installed
E: Unable to correct problems, you have held broken packages.

``` 

### System information

```shell
Termux Variables:
TERMUX_API_APP__VERSION_NAME=0.50.1+8651fd9
TERMUX_APP_PACKAGE_MANAGER=apt
TERMUX_APP__AM_SOCKET_SERVER_ENABLED=true
TERMUX_APP__APK_PATH=/data/app/~~6Z9LBhRHojH7D6ORdoSiBg==/com.termux-H8tjtvsrFauigMCMUQ5FPg==/base.apk
TERMUX_APP__APK_RELEASE=GITHUB
TERMUX_APP__FILES_DIR=/data/user/0/com.termux/files
TERMUX_APP__IS_DEBUGGABLE_BUILD=true
TERMUX_APP__IS_INSTALLED_ON_EXTERNAL_STORAGE=false
TERMUX_APP__PACKAGE_MANAGER=apt
TERMUX_APP__PACKAGE_NAME=com.termux
TERMUX_APP__PACKAGE_VARIANT=apt-android-7
TERMUX_APP__PID=32157
TERMUX_APP__SE_FILE_CONTEXT=u:object_r:app_data_file:s0:c222,c257,c512,c768
TERMUX_APP__SE_INFO=default:targetSdkVersion=28:complete
TERMUX_APP__SE_PROCESS_CONTEXT=u:r:untrusted_app_27:s0:c222,c257,c512,c768
TERMUX_APP__TARGET_SDK=28
TERMUX_APP__UID=10478
TERMUX_APP__USER_ID=0
TERMUX_APP__VERSION_CODE=118
TERMUX_APP__VERSION_NAME=0.118.0+e59e5c1
TERMUX_MAIN_PACKAGE_FORMAT=debian
TERMUX_VERSION=0.118.0+e59e5c1
TERMUX__USER_ID=0
Packages CPU architecture:
aarch64
Subscribed repositories:
# sources.list
deb https://packages-cf.termux.org/apt/termux-main stable main
# tur-repo (sources.list.d/tur.list)
deb https://tur.kcubeterm.com tur-packages tur tur-on-device tur-continuous
# x11-repo (sources.list.d/x11.list)
deb https://packages-cf.termux.dev/apt/termux-x11/ x11 main
Updatable packages:
All packages up to date
termux-tools version:
1.44.1
Android version:
14
Kernel build information:
Linux localhost 5.15.123-android13-8-29539737-abX916NKOU4BXHB #1 SMP PREEMPT Fri Aug 30 12:32:27 UTC 2024 aarch64 Android
Device manufacturer:
samsung
Device model:
SM-X916N
LD Variables:
LD_LIBRARY_PATH=
LD_PRELOAD=/data/data/com.termux/files/usr/lib/libtermux-exec.so
Installed termux plugins:
com.termux.widget versionCode:13
com.termux.x11 versionCode:14
com.termux.api versionCode:51
```

**Comment by truboxl** [comment]
https://github.com/termux/termux-packages/pull/21843

**Comment by twaik** [comment]
Fixed by ebee2c0be2e380881149adad8580491d11390b0d.
